SignalServiceKit/src/Messages/Attachments/OWSAttachmentsProcessor.m

```diff
@@ -35,37 +35,30 @@
 @interface OWSAttachmentsProcessor ()
 
 @property (nonatomic, readonly) TSNetworkManager *networkManager;
-@property (nonatomic, readonly) OWSPrimaryStorage *primaryStorage;
-@property (nonatomic, readonly) NSArray<TSAttachmentPointer *> *supportedAttachmentPointers;
```

Comment:
"supportedAttachmentPointers" is a legacy field from when we didn't support arbitrary attachments.